tools/xtce_generation/xtce_generator_lo.py

```diff
@@ -0,0 +1,50 @@
+from pathlib import Path
+
+from telemetry_generator import TelemetryGenerator
+
+
+def main():
+    """This function can be used by any instrument to generate XTCE
+    for certain number of packets. Change values where TODO is
+    """
+    instrument_name = "lo"
+    current_directory = Path(__file__).parent
+    module_path = f"{current_directory}/../../imap_processing"
+    packet_definition_path = f"{module_path}/{instrument_name}/packet_definitions"
+    path_to_excel_file = f"{current_directory}/telem_def_lo_modified.xls"
```

Comment:
Did you document how you had to modify this / update it? It might be good to put a few comments in here or somewhere so we have it documented what you did to this file (if anything).

Reply:
Do you mean document how I updated the lo xtce generator file or how I modified the XTCE files after they were generated?

Reply:
Both? I am just trying to help our future selves when we come back to this in 6 months and forget the exact steps you did to go from the xcel sheet to the files in this PR, so it'd be good to document it here for the next time we get the Lo deliveries.